AndroidDeveloperVerificationOnboarding/app/src/androidTest/java/com/androiddeveloperverification/onboarding/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Remove file and package if you're not using

Reply:
done. Thanks!